tests/pax/common.libsonnet

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local common = import '../common.libsonnet';
+local experimental = import '../experimental.libsonnet';
+local mixins = import 'templates/mixins.libsonnet';
+local tpus = import 'templates/tpus.libsonnet';
+
+{
+  PaxTest:: common.CloudAcceleratorTest + experimental.BaseTpuVmMixin {
```

Comment:
I know it's clunky, but you should include some sort of base command that cleans up the TPU after the test script runs (`/scripts/cleanup.sh`). JAX is probably the closest example:

https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/20c22a3517804809ba60c48354043937a11447b7/tests/jax/common.libsonnet#L64-L85